CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 - `realm-management {interfaces, triggers} sync`: do not show a wrong curl
   command when using `--to-curl`. Reference individual commands instead.
+- `appengine {publish-datastream set-property}`: fix the decoding of longinteger
```

Comment:
The change seems to be only inside `devicesPublishDataStreamF`, could you check if it is in `set-property` too?

Reply:
Edit: it makes sense, as aggregated properties do not exist